cmd/top-api-spinner.go

```diff
@@ -174,15 +178,23 @@ func (m *traceUI) View() string {
 	for k, stats := range m.apiStatsMap {
 		data = append(data, []string{
 			k,
-			whiteStyle.Render(humanize.IBytes(stats.loadAPIBytesRX())),
```

Comment:
Is removal of coloring expected here ?

Reply:
You should sort this before if necessary, otherwise it's simply expensive to parse the strings again and again.